README.md

```diff
@@ -53,7 +61,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+<!-- In line 58, = is the assignment value of sunny, weather = in ths case can change depending on factors that would change the value.  In line 59, weather === sunny is an absolute value for the variable weather, and would either evaluate to true or false. -->
```

Comment:
Not quite right.
The first line creates a variable `weather`; at this point, its value is `undefined`. The second line assigns the value "sunny" to weather. The third line is a comparison operation; if `weather` is equal to "sunny", it will evaluate to `true`.